src/bindgen/ir/documentation.rs

```diff
@@ -39,25 +39,57 @@ impl Source for Documentation {
             return;
         }
 
-        if config.language == Language::C {
-            out.write("/**");
-            out.new_line();
+        let style = match &config.documentation_style {
```

Comment:
Just `match config.documentation_style`, and then you can remove `*other` and just use `other`?

Reply:
Yes, done, thanks for review!